SachinSinghDighan/RecipeRoulette/app/src/main/java/com/example/reciperoulette/di/Qualifiers.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.reciperoulette.di
+
+import javax.inject.Qualifier
+
+@Qualifier
```

Comment:
what is the use of these qualifiers?

Reply:
@nasreekar qualifiers are used by dagger to distinguish between two variables with same data type, suppose there is method in module we have two methods which returns different URL for development and for QA since their return type is same which is string so let dagger know which method return development url  and which return other url for that we use qualifiers and annotate that method or variable with qualifier.